Source/missiles.cpp

```diff
@@ -2841,6 +2841,7 @@ int AddMissile(Point src, Point dst, int midir, missile_id mitype, mienemy_type
 	missile._mitype = mitype;
 	missile._micaster = micaster;
 	missile._misource = id;
+	missile._midam = midam;

```

Comment:
Was this removed by mistake at some point?

Reply:
Yeah https://github.comdiasurgical/devilutionX/pull/2676/files#diff-cf201edb86708433771f232de909a54b7712af8ed1e9feeadd25d149a8e9caf7L2916